react-components/src/components/MyAccount/myAccount.js

```diff
@@ -39,12 +39,20 @@ const MyAccount = props => {
         );
     }
 
+    const redirectBacktoStorefront = () => {
+        window.location.href = pagePaths.baseUrl;
+    };
+
     const handleSignOut = async () => {
         dispatch({ type: 'reset' });
         await signOut();
         if (showMenu) {
             showMenu();
         }
+
+        if (pagePaths && pagePaths.baseUrl) {
```

Comment:
I would move the check `pagePaths && pagePaths.baseUrl` into the `redirectBacktoStorefront` method at line 42, so you check the variable immediately before usage.

Reply:
Good call. Moved.